packages/node_modules/@webex/internal-plugin-wdm/README.md

```diff
@@ -6,12 +6,17 @@
 
 This is an internal Cisco Webex plugin. As such, it does not strictly adhere to semantic versioning. Use at your own risk. If you're not working on one of our first party clients, please look at our [developer api](https://developer.webex.com/getting-started.html) and stick to our public plugins.
 
+- [Deprecation](#deprecation)
 - [Install](#install)
 - [Usage](#usage)
 - [Contribute](#contribute)
 - [Maintainers](#maintainers)
 - [License](#license)
 
+## Deprecation
```

Comment:
Simple plugin deprecation message, any feedback on how this should appear is welcome.

Reply:
I think this works for now since we are still planning on supporting users using the `wdm` plugin. 

Once we completely wipe it out, let's use the readme template from one of the `@ciscospark` plugins that points to the new folder.